hub/integrations/wandb/wandb.py

```diff
@@ -57,6 +58,8 @@ def artifact_from_ds(ds):
     path = ds.path
     name = artifact_name_from_ds_path(ds)
     artifact = wandb.Artifact(name, "dataset")
+    if '://' not in path and os.path.exists(path):
+        path = 'file://' + path
```

Comment:
@hakanardo Alternatively you can check if a dataset is a local dataset like this:

```suggestion
if isinstance(ds.base_storage, LocalProvider):
    path = "file://" + path
```

Reply:
Nice!